internal/setup/yaml.go

```diff
@@ -0,0 +1,464 @@
+// This file contains the YAML-specific logic needed to support setup.go.
+
```

Comment:
Only nitpick, let's remove this line, I don't see it done anywhere else in the code.